tests/test_AggregateWellData.cpp

```diff
@@ -651,7 +666,6 @@ BOOST_AUTO_TEST_CASE (Declared_Well_Data)
         BOOST_CHECK_EQUAL(iwell[start + Ix::CompOrd], 0); // Track ordering (default)
 
         BOOST_CHECK_EQUAL(iwell[start + Ix::item18], -100); // M2 Magic
-        BOOST_CHECK_EQUAL(iwell[start + Ix::item25], -  1); // M2 Magic
```

Comment:
where did the check go? item25 is now WGrupConControllable, but that seems not to be used in the tests.

Reply:
> where did the check go?

We used to hard-code this value to `-1` due to "M2 magic", but that is no longer the case.

> item25 is now `WGrupConControllable`, but that seems not to be used in the tests.

We don't have a test case that exercises this setting.  I'll see about adding one.